pytest_timeout.py

```diff
@@ -39,58 +38,61 @@
 used in the test.
 """.strip()
 
-Settings = namedtuple('Settings', ['timeout', 'method', 'func_only'])
+Settings = namedtuple("Settings", ["timeout", "method", "func_only"])
 
 
 @pytest.hookimpl
 def pytest_addoption(parser):
     """Add options to control the timeout plugin"""
     group = parser.getgroup(
-        'timeout', 'Interrupt test run and dump stacks of all threads after '
-        'a test times out')
-    group.addoption('--timeout',
-                    type=float,
-                    help=TIMEOUT_DESC)
-    group.addoption('--timeout_method',
-                    action='store',
-                    choices=['signal', 'thread'],
-                    help='Deprecated, use --timeout-method')
-    group.addoption('--timeout-method',
-                    dest='timeout_method',
-                    action='store',
-                    choices=['signal', 'thread'],
-                    help=METHOD_DESC)
-    parser.addini('timeout', TIMEOUT_DESC)
-    parser.addini('timeout_method', METHOD_DESC)
-    parser.addini('timeout_func_only', FUNC_ONLY_DESC, type='bool')
+        "timeout",
+        "Interrupt test run and dump stacks of all threads after " "a test times out",
```

Comment:
I assume this is black annoying old people by completely ignoring 79 char limits... but that's a pretty poor version of this.  Could you turn this back into a single string at least?

Reply:
```suggestion
        "Interrupt test run and dump stacks of all threads after a test times out",
```